modules/base/markdown.go

```diff
@@ -346,7 +340,7 @@ OUTER_LOOP:
 func RenderMarkdown(rawBytes []byte, urlPrefix string, metas map[string]string) []byte {
 	result := RenderRawMarkdown(rawBytes, urlPrefix)
 	result = PostProcessMarkdown(result, urlPrefix, metas)
-	result = Sanitizer.SanitizeBytes(result)
+	result = BuildSanitizer().SanitizeBytes(result)
```

Comment:
As mentioned in #2396, this should only build once (at start up) and reuse the same object `p`.